doc/configuration.rst

```diff
@@ -957,41 +957,42 @@ you can do::
 Controlling what output is captured
 ===================================
 
-Sphinx-Gallery can run your code blocks in two ways:
+The ``print_eval_repr`` configuration allows the user to control what output of
+the last expression in a code block, is captured and rendered in the built
+documentation. By default ``print_eval_repr`` is an empty tuple. With this
+setting, the last statement in a code block is executed with ``exec()`` and
```

Comment:
How this is accomplished (`eval` vs `exec`) is an implementation detail SG users don't need to know/care about. I would just describe that:
```
x = pandas.DataFrame(...)
```
is never rendered, but:
```
x
```
will print depending on the value of `print_eval_repr`.

Reply:
> How this is accomplished (eval vs exec) is an implementation detail SG users don't need to know/care about. 

You're right and I was thinking this when I wrote it.

> will print depending on the value of print_eval_repr.

This is a better explanation. The problem is that I have not implemented capturing `_repr_html_` yet and I was trying to make this PR and thus the documentation in it, applying only to the 'exec/eval'.

Reply:
If you want, you can make the docs for now just about `()` and `('__repr__',)`, and add `__html_repr__` support separately

Reply:
@lucyleeow don't forget to address this